docs/types.rst

```diff
@@ -41,6 +41,8 @@ Operators:
 
 The operators ``||`` and ``&&`` apply the common short-circuiting rules. This means that in the expression ``f(x) || g(y)``, if ``f(x)`` evaluates to ``true``, ``g(y)`` will not be evaluated even if it may have side-effects.
 
+For more information about logical and relational operators (as used in C++), see `here <http://www.learncpp.com/cpp-tutorial/36-logical-operators/>`_ and `here <http://www.learncpp.com/cpp-tutorial/35-relational-operators-comparisons/>`_, respectively. 
```

Comment:
Hmm I guess that's better than nothing but ideally there would be another version of the docs that educates how to program in Solidity in detail.